lib/diagnostic.js

```diff
@@ -39,20 +49,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Blitz',
+  species: 'cat',
+  breed: 'flame-point himmy',
+  vocalization: 'loud meows'};
 
```

Comment:
We wanted to add a method directly here called `vocalize`, that would look something like this: `function() {return this.vocalization}`